lib/management/types.ts

```diff
@@ -922,3 +922,33 @@ export type InboundApplicationConsentDeleteOptions = {
   appId?: string;
   userIds?: string[];
 };
+
+export type PromptType = 'none' | 'login' | 'consent' | 'select_account';
+export type AccessType = 'offline' | 'online';
```

Comment:
since its exported i would probably rename it
```suggestion
export type OutboundApplicationAccessType = 'offline' | 'online';
```